<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-5PE8I9M0" ignored="0" name="Test" tests="2" failures="2" timestamp="2022-07-25T19:07:43 IST" time="99.606" errors="0">
  <testcase name="login" time="18.95" classname="com.mystore.testCases.LoginPageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.mystore.testCases.LoginPageTest.login(LoginPageTest.java:39)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase name="loginFailed" time="13.048" classname="com.mystore.testCases.LoginPageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.mystore.testCases.LoginPageTest.loginFailed(LoginPageTest.java:67)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- loginFailed -->
</testsuite> <!-- Test -->
